pkgs/development/compilers/gerbil/unstable.nix

```diff
@@ -1,13 +1,13 @@
 { stdenv, callPackage, fetchgit, gambit-unstable }:
 
 callPackage ./build.nix {
-  version = "unstable-2018-09-06";
-  git-version = "0.14-DEV";
+  version = "unstable-2018-11-19";
+  git-version = "0.15-DEV";
   GAMBIT = gambit-unstable;
   SRC = fetchgit {
```

Comment:
Why not use `fetchFromGitHub`?

Reply:
Let me try again that way...

Reply:
It builds, so I amended things your way. done.